src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +70,7 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    //default isMutating method with return value of true
```

Comment:
Implementing `isMutating()` in this class is not sufficient to achieve the polymorphic behavior specified in this LO. To do so, you would have to implement this in the other `Command` classes and return true or false respectively for each class. Then, you would have to do a check in Logic and that is where the polymorphic behavior happens. 

Having a default implementation for the isMutating method helps to reduce the amount of code. However, some may argue that choosing false (or true) as the default is arbitrary and hurts the intuitiveness of the code. Making the method abstract forces all child classes to declare whether it is a mutating command or not, rather than depend on an arbitrary decision made in the parent class.